### ISSUE TITLE
Search modal should auto focus

### ISSUE THREAD

**Issue by damiensedgwick** [fireship-io/fireship.io#1487]
When pressing `/` to open the search modal, I still have to grab my mouse to click inside of the search bar before I can start typing.

It would be smoother if I could press `/` and start typing right away.

**Comment by joeljtomy** [comment]
#1647 this PR will fix the problem

- simply puts a timeout of 50 ms when focusing the element and it works
- use action
preview:

![ezgif com-video-to-gif (1)](https://github.com/fireship-io/fireship.io/assets/122166595/62aaa50a-da35-4adf-856b-88e517ffe786)

**Comment by damiensedgwick** [comment]
> #1647 this PR will fix the problem
> 
> * simply puts a timeout of 50 ms when focusing the element and it works
> * use action
>   preview:
> 
> ![ezgif com-video-to-gif (1)](https://user-images.githubusercontent.com/122166595/260258958-62aaa50a-da35-4adf-856b-88e517ffe786.gif)

Nice! Does this work the same way when using the `/` shortcut?

**Comment by joeljtomy** [comment]
@damiensedgwick yes it does whenever the input is mounted to the dom focus()  function is called on input element
